src/test/java/bio/terra/service/filedata/google/firestore/FileLoadTest.java

```diff
@@ -135,6 +137,43 @@ public void fileLoadTest() throws Exception {
 
         assertThat(summary.getSucceededFiles(), equalTo(filesToLoad));
     }
+
+    // The purpose of this test is to have a long-running workload that completes successfully
+    // while we delete pods and have them recover.
+    @Test
+    public void longFileLoadTest() throws Exception {
+        // TODO: want this to run about 5 minutes on 2 DRmanager instances. The speed of loads is when they are
+        //  local is about 2.5GB/minutes. With a fixed size of 1GB, each instance should do 2.5 files per minute,
```

Comment:
Typo? Speed of loads when the files are Not local is ~2.5GB/min

Reply:
My memory is clearly shot... I thought the NOT local case was around 1.5GB/minute and the local case was around 2.5GB. Please refresh my memory on the proper numbers?